airflow/api_connexion/openapi/v1.yaml

```diff
@@ -3442,6 +3449,11 @@ components:
         - sensing
         - removed
 
+    NullableTaskState:
+      oneOf:
+        - type: null
+        - $ref: '#/components/schemas/TaskState'
+
```

Comment:
Should we have this immediately after TaskInstance which is where it was used?

Reply:
I kind of feel we might be better off expanding this in-place and get rid of the new components entirely.

Reply:
1. TaskState/TimeDelte itself can not be null, we couldn't change itself.  It is not nullable in some scenarios, like:
```
# 3259~3264
state:
          type: array
          items:
            $ref: '#/components/schemas/TaskState'
          description:
            The value can be repeated to retrieve multiple matching values (OR condition).
```

2. As discussed in [openapi-generator](https://github.com/OpenAPITools/openapi-generator/issues/11352), the inline composed schemas has bug using tooling to generate client code. If you expect a property to be one of TaskState/TimeDelta and null, the following code also doesn't work:
```
dag_run_timeout:
 oneOf:
  - type: null
  - $ref: '#/components/schemas/TimeDelta'
```

3. Of course we could wait for the `openapi-generator` to fix the bug, but not necessarily in time. It's a trade-off.

Reply:
I could change it into inline composed schemas to fix the current `wrong composed schemas` bug, instead of creating a new component. But it is still not working.